tensorflow/compiler/tf2tensorrt/convert/ops/selectv2.cc

```diff
@@ -91,30 +113,86 @@ class ConvertSelectV2 : public OpConverterBase<ConvertSelectV2> {
         *tensor_[2]->trt_tensor()   // else_tensor
     );
 
-    converter->SetLayerName(select_layer, params.node_def.name(), "selectv2");
+    converter->SetLayerName(select_layer, params.node_def.name(), layer_name_);
     AddOutput(TRT_TensorOrWeights(select_layer->getOutput(0)));
     return OkStatus();
   }
 
  private:
+  bool compare_shapes(const nvinfer1::Dims& shape1,
+                      const nvinfer1::Dims& shape2,
+                      bool then_vs_else) const {
+    bool same_shapes = shape1 == shape2;
+    if (!same_shapes && shape1.nbDims == shape2.nbDims) {
+      // The shapes still should be considered as equal if at least one of
+      // them is a tensor with dynamic shape
```

Comment:
Please add a period to the end of the comment. Would it be better to revise the comment to this:
We can't check size equivalent when dynamic shapes are involved. In this case, the two shapes should be equal at runtime. Otherwise, <put the expected behavior here>.

Reply:
Done